packages/ui/src/CodeEditor.tsx

```diff
@@ -147,13 +185,26 @@ export const CodeEditor: FC<CodeEditorProps> = ({ value, className, options = {}
         ]),
       ]
 
+      const fixedHeightExtension = () => {
+        if (!opts.rows) return []
+
+        const LINE_HEIGHT = 22
+        const h = LINE_HEIGHT * opts.rows + 4
```

Comment:
nit: what does the `+ 4` stand for? Some free space at top and bottom?

Reply:
yep, and let me make that a constant.